src/main/java/io/github/moulberry/notenoughupdates/NotEnoughUpdates.java

```diff
@@ -1008,7 +1008,8 @@ public void preinit(FMLPreInitializationEvent event) {
             } catch(Exception e) { }
         }
 
-        FairySouls.load(new File(neuDir, "collected_fairy_souls.json"), gson);
+        FairySouls.load(neuDir, gson);
+        Relics.load(neuDir, gson);
```

Comment:
why do u not use `(neuDir, "collected_relics.json")` here?

Reply:
makes sense

Reply:
do you happen to have a working jar?

Reply:
Yes, but I would prefer not to start sharing private versions of the JAR since I don't want to create support problems for Moulberry and crew.

Reply:
funny thing is the way u did this won't work since you forgot to fix 1 line ;)